modules/ROOT/attachments/admin-api.yaml

```diff
@@ -1049,7 +1049,35 @@ paths:
         required: false
         schema:
           type: integer
-          format: int64 
+          format: int64
+  /debug/partitions/{namespace}/{topic}/{partition}/force_replicas:
+    post: 
+      tags: 
+      -  Debugging
```

Comment:
This is more related to "fixing" than "debugging" (i.e. understanding what went wrong). Therefore I think this should be a separate tag named something like "escape hatches" or "disaster recovery" if we have it.

Reply:
Maybe a better place for now is 'Partitions'